### DISCUSSION TITLE
Chinese characters are shown as dots

### DISCUSSION THREAD

**Discussion by KyQiao** [focus-editor/focus#453] [Q&A] [answered]
Hi. I really enjoy Focus's design philosophy and wish to continue using it. Yet, I find it cannot properly show Chinese characters, which I believe are not multi-codepoint Unicode symbols. I tried another font like Cascadia Code and still got the dot in the picture. Any solution for fixing it? Thanks.
![图片](https://github.com/user-attachments/assets/af4d4950-ceab-45ff-aa24-f3100bd76006)

**Comment by focus-editor** [comment, marked as answer]
Hi. Unfortunately we don't support CJK characters properly yet. The editor is not 100% done.
If a font which has Chinese characters is used it _should_ display them, but the spacing would be off, so it's not actually usable.
We will add support for it at some point, but only after the majority of things are sorted out. While we're still figuring things out we're only focusing on western languages.
